daemons/controld/controld_te_callbacks.c

```diff
@@ -89,13 +89,13 @@ te_update_diff_v1(const char *event, xmlNode *diff)
     for (lpc = 0; lpc < max; lpc++) {
         xmlNode *attr = getXpathResult(xpathObj, lpc);
         const char *name = crm_element_value(attr, XML_NVPAIR_ATTR_NAME);
-        const char *value = NULL;
+        bool value = false;
 
         if (pcmk__str_eq(CRM_OP_PROBED, name, pcmk__str_casei)) {
-            value = crm_element_value(attr, XML_NVPAIR_ATTR_VALUE);
+            value = pcmk__xe_get_bool_attr(attr, XML_NVPAIR_ATTR_VALUE);
         }
 
-        if (crm_is_true(value) == FALSE) {
+        if (!value) {
```

Comment:
You could avoid the variable with if (... && ...)

Reply:
Fixed.